Updates to wording in docs (#281)
opensafely-core/airlock: docs/index.md

Researchers use the Airlock UI to view their workspace files, create a new release request, add their files to the release and submit them for review. Researchers can only have at most one open release request per workspace, but files can be organised by the researchers into logical groups when constructing the request.

> Output checkers are able to review submitted release requests, including viewing the files within the request, approving or rejecting these files. They can then approve or reject the request as a whole, and finally release the files to the job-server.

Reviewer: ```suggestion
They can then either reject the request as a whole or release the files to the job-server (implicitly approving the request as a whole).
```

Wdyt - bit mealy-mouthed, but describes the current UI - which just has buttons for "reject" or "release", and doesn't have one for "approve"?

Reviewer 2: This seems clear to me, as a potential output checker
